src/dotnet/APIView/APIViewWeb/Managers/PullRequestManager.cs

```diff
@@ -150,6 +151,11 @@ public async Task<string> CreateAPIRevisionIfAPIHasChanges(
                 baselineCodeFileName: baselineCodeFileName, baselineStream: baselineStream,
                 project: project, language: language);
 
+            if (_codeFileManager.AreLineIdsDuplicate(codeFile, out string duplicateLineId))
```

Comment:
So I see where this check is done for PR and Automatic revisions, but what about manual? We want to ensure (ideally in one place if possible) that any revision to APIView is checked.

Reply:
oh I though we were targeting pipelines, changed this.

Reply:
Nope, as there are languages that don't have CI-generated reviews. It should fail in CI, yes, but a manual review should not surface a generic error that prompts someone to bug you 👍